### ISSUE TITLE
v0.8.4+ breaks Android 4.1 support

### ISSUE THREAD

**Issue by Coolcat13** [katzer/cordova-plugin-badge#106]
Hi there, just to let you know, from 0.8.4 this plugin doesn't work with Android 4.1 (works ok on 4.4 - no device to test on 4.2 and 4.3) - I downgraded to v 0.8.3 for now.

The error is: Object function Object() { [native code] } has no method 'assign' in cordova.js line 315.

Hope this helps
Thanks!

**Comment by ferdelcorro** [comment]
I have the same error with Android 4.4.2, 4.4.4 and 5.0

```
12-04 09:42:35.989 I/Choreographer( 1533): Skipped 31 frames!  The application may be doing too much work on its main thread.
12-04 09:42:36.129 I/chromium( 1533): [INFO:CONSOLE(312)] "Error in Success callbackId: Badge1665621330 : TypeError: Object function Object() { [native code] } has no method 'assign'", source: file:///android_asset/www/cordova.js (312)
12-04 09:42:36.129 I/chromium( 1533): [INFO:CONSOLE(314)] "Uncaught TypeError: Object function Object() { [native code] } has no method 'assign'", source: file:///android_asset/www/cordova.js (314)
```

**Comment by katzer** [comment]
Should be fixed now with v0.8.6
